src/TriggerActivityMakerDBSCAN.cpp

```diff
@@ -60,17 +55,16 @@ TriggerActivityMakerDBSCAN::operator()(const TriggerPrimitive& input_tp, std::ve
       ta.adc_integral += prim.adc_integral;
 
       ta.detid = prim.detid;
+      if (prim.adc_peak > ta.adc_peak) {
+        ta.adc_peak = prim.adc_peak;
+        ta.channel_peak = prim.channel;
+        ta.time_peak = prim.time_peak;
+      }
     }
```

Comment:
Not for this PR, but I imagine that sometimes adding a TP will not add anything to a cluster, so maybe `m_scan` having some form of `get_status_changed` function, and only doing this double-for after a `yes` could speed things up a bit.